_posts/2020-01-13-no-app-bundler-needed.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "No app bundler needed for your next project?"
+date: 2020-01-12 06:00:12 +0000
+permalink: /coding/no-app-bundler-needed
+category: coding
+published: true
+full_image_url: "https://user-images.githubusercontent.com/10452163/71785934-7981c300-2ffd-11ea-8cfd-4fc2ae8c74a3.jpg"
+meta_description: >
+  Exploring new language features supported in the browser without a bundler.
+excerpt_separator: <!--more-->
+tags: javascript
+---
+
+The browsers are rolling out updates to support more of the latest features of JavaScript defined by _ECMAScript_ [technical committee 39](https://github.com/tc39){:target="\_blank" rel="noopener"}. Have you thought about how much can we write today without using an app bundler like [Webpack](https://webpack.js.org/){:target="\_blank" rel="noopener"}, [Rollup.js](https://rollupjs.org/guide/en/){:target="\_blank" rel="noopener"} or [Parcel](https://parceljs.org/){:target="\_blank" rel="noopener"}? Below I will go through a few _JavaScript features_ we can use in today's modern web browser when building a new web app.
+
+<!--more-->
+
+It may not be a surprise but a lot of the features are **not** supported by _Internet Explorer_ but _Edge_ seems to do a decent job.
```

Comment:
Unsurprisingly, _Internet Explorer_ doesn't support many of these features, however, _Edge_ seems to do a decent job.